src/Admin_Tools_Command.php

```diff
@@ -100,6 +100,8 @@ public function install() {
 	 */
 	public function up( $args, $assoc_args ) {
 
+		\EE\Auth\Utils\init_global_admin_tools_auth();
```

Comment:
How do you know this function will always be accessible? There is no [composer dependency](https://github.com/EasyEngine/admin-tools-command/blob/develop/composer.json) also. Please check if `method_exists( '\EE\Auth\Utils\init_global_admin_tools_auth' )` then procced next. 

Since this is a required step you can throw error saying that auth package is required when this method is not exists.

Reply:
I think it makes more sense to include auth-command in dependency since we always enable auth on admin-tools by default. So instead of making explicit check, we include auth-command implicitly by adding it in composer dependeny